patchwork/models.py

```diff
@@ -729,6 +736,15 @@ class Meta:
         ]
 
 
+class PatchReviewIntention(models.Model):
```

Comment:
I quite like the "attention set" term since it allows us to capture both a review intention and a review request. Can we use it?

```suggestion
class PatchAttentionSet(models.Model):
```